examples/js/controls/OrbitControls.js

```diff
@@ -79,6 +76,9 @@ THREE.OrbitControls = function ( object, domElement ) {
 	this.position0 = this.object.position.clone();
 	this.zoom0 = this.object.zoom;
 
+	// the target DOM element for key events
+	this._domElementKeyEvents = null;
```

Comment:
Hmm, does it need to be public?

Reply:
The object needs to be accessible in `dispose()`. With the current structure of `OrbitControls` (and also when using classes), I don't think there is another solution.